src/transformers/Streamable.js

```diff
@@ -0,0 +1,32 @@
+import { URL } from 'url';
+import fetch from 'node-fetch';
+
+const getTrimmedPathName = pathname => pathname.replace(/^\/|\/+$/g, '');
+
+const includesSomeOfArray = (string, array) =>
+  array.some(item => string.includes(item));
+
+export const shouldTransform = url => {
+  const { host, pathname } = new URL(url);
+  const trimmedPathName = getTrimmedPathName(pathname).split('/');
+
+  return (
+    ['streamable.com', 'www.streamable.com'].includes(host) &&
+    !includesSomeOfArray(pathname, [
```

Comment:
Only check if `pathname` is included in array when `trimmedPathName.length === 1`